runtime/common/src/main/kotlin/kotlinx/serialization/modules/SerialModuleExtensions.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+@file:Suppress("RedundantVisibilityModifier")
+/*
+ * Copyright 2018 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kotlinx.serialization.modules
+
+import kotlinx.serialization.*
+import kotlin.reflect.KClass
+
+/**
+ * Returns a dependent serializer associated with a given reified type.
+ *
+ * This method is used in context-sensitive operations
```

Comment:
General note: thing like "this method is used in ..." is an implementation detail and usually not worth mentioning.